components/ProfilePageComponent/ProfileBasicInfo.tsx

```diff
@@ -0,0 +1,187 @@
+    flex-wrap: wrap;
+    gap: 8px;
+`;
+
+const UserNameParagraph = Styled.p`
+    font-size: 42px;
+    line-height: 1.15;
+    font-weight: 600;
+    color: inherit;
+    margin: 0px;
+`;
+
+const UserDescriptionAndSocialContainer = Styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+`;
+
+const UserDescriptionContainer = Styled.div`
+    margin-right: auto;
+    width: 50%;
+`;
+
+const UserDescriptionParagraph = Styled.p`
+    font-size: 24px;
+    line-height: 1.15;
+    font-weight: 400;
+    color: inherit;
+    margin: 0px;
+`;
+const SocialIconContainer = Styled.div`
+    width: 20%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+`;
+
+const UserLocationAndPhoneContainer = Styled.div`
+    display: flex;
+    gap: 8px 88px;
+    margin-top: 5px;
+`;
+
+const UserLocationAndPhone = Styled.div`
+    display: flex;
+    gap: 2px;
+`;
+
+const UserLocationAndPhoneParagraph = Styled.div`
+    font-size: 16px;
+    line-height: 1.15;
+    font-weight: 400;
+    color: inherit;
+`;
+
+const UserPrivacyNoteContainer = Styled.div`
+    margin-bottom: -8px;
+    background-color: rgb(223, 236, 249);
+    border-radius: 4px;
+    color: #1f2228;
+    padding: 16px;
+    margin: 0px 48px;
+`;
+
+const UserPrivacyNoteParagraphContainer = Styled.div`
+    display: flex;
+    // justify-content: center;
+    align-items: center;
+    gap: 12px;
+`;
+
+const PrivacyNoteParagraph = Styled.div`
+    font-size: 14px;
+    line-height: 1.35;
+    font-weight: 600;
+    color: inherit;
+    margin: 0px;
+`;
+const PrivacyNoteDescriptionParagraph = Styled.div`
+    font-size: 14px;
+    line-height: 1.35;
+    font-weight: 400;
+    color: inherit;
+    margin: 0px;
+`;
+const ProfileBasicInfo = () => {
+    return (
+        <Container>
+            <HeaderContainer />
+            <UserProfilePic />
+            <div
+                style={{
+                    margin: '0px 0px 8px',
+                    height: '32px',
+                }}
+            ></div>
+            <UserInformationContainer
+                style={{
+                    display: 'flex',
+                    flexDirection: 'column',
+                    alignItems: 'flex-start',
+                    gap: '12px',
+                    marginLeft: '48px',
+                }}
+            >
+                <UserNameContainer>
+                    <UserNameParagraph>Muhammad Hussain</UserNameParagraph>
+                </UserNameContainer>
+                <UserDescriptionAndSocialContainer>
+                    <UserDescriptionContainer>
+                        <UserDescriptionParagraph>
+                            Junior Full stack developer Focused on Frontend, Skilled in Javascript,
+                            React and Node.Js
+                        </UserDescriptionParagraph>
+                    </UserDescriptionContainer>
+                    <SocialIconContainer>
```

Comment:
@fazlerabby07 we have font-awesome installed. why don't you pull and use font awesome icons instead?

Reply:
@sifatul okey i will.

Reply:
@sifatul fixed it please check.